src/vicsetup/rules.mak

```diff
@@ -0,0 +1,28 @@
+OBJECTS = $(SOURCES:.c=.o)
+
+ifdef PROGRAM
+$(PROGRAM): dirs $(OBJECTS)
+	gcc -o $(PROGRAM) $(CFLAGS) $(OBJECTS) $(LDFLAGS)
```

Comment:
Don't hard-code the compiler, use CC.

Reply:
Fixed: Good catch. Thanks for finding this.